packages/studio-ui/src/components/EditorSidebar.tsx

```diff
@@ -25,16 +26,18 @@ export default function EditorSidebar(): JSX.Element {
   const [selectedTab, setTab] = useState<Tab>(Tab.Props);
 
   return (
-    <div className="w-1/4 px-4 flex flex-col" data-testid="EditorSidebar">
-      <OptionPicker
-        options={Tab}
-        icons={tabIcons}
-        defaultOption={Tab.Props}
-        onSelect={setTab}
-      />
-      <Divider />
-      {renderTab(selectedTab)}
-    </div>
+    <CollapsibleSidebar side="right">
+      <div className="flex flex-col grow px-4" data-testid="EditorSidebar">
```

Comment:
would we want this data attribute to be on `CollapsibleSidebar` now?

Reply:
moved it back down since we figured out that the`data-testid` does not play well with react component props.